documentation/modules/adding-source-provider.adoc

```diff
@@ -50,8 +50,8 @@ ifdef::vmware[]
 
 * *Name*: Name to display in the list of providers
 * *Hostname or IP address*: vCenter host name or IP address
-* *Username*: vCenter admin user, for example, `administrator@vsphere.local`
-* *Password*: vCenter admin password
+* *Username*: vCenter user, for example, `user@vsphere.local`
+* *Password*: vCenter user password
```

Comment:
Please remove the 'admin' also from "Create a secret from CLI" (a secret resource is needed for the Provider)
![image](https://user-images.githubusercontent.com/60384172/153810445-a177d821-b7d7-4277-932e-66e2284676fb.png)